libs/core/langchain_core/agent_toolkits.py

```diff
@@ -0,0 +1,14 @@
+"""Toolkits for agents."""
+from abc import ABC, abstractmethod
+from typing import List
+
+from langchain_core.pydantic_v1 import BaseModel
+from langchain_core.tools import BaseTool
+
+
+class BaseToolkit(BaseModel, ABC):
```

Comment:
@baskaryan should we group this into the tools namespace instead? nothing about a tool kit is agent specific

Reply:
yep would put in tools.py